docs/getting-started/croeseid-testnet-nix.md

```diff
@@ -0,0 +1,69 @@
+# Croeseid Testnet: Running Nodes With Nix
+
+## Pre-requisites
+
+Nix supports both linux and mac, for windows user, you can follow the [manual instructions](./croeseid-testnet.md).
+
+## Install nix
+
+Follow official instructions to install nix: https://nixos.org/download.html
+
+Install cachix and enable crypto-com binary cache:
+
+```shell
+$ nix-env -iA cachix -f https://cachix.org/api/v1/install
+$ cachix use crypto-com
+```
+
+## Install chain utils for testnet
+
+```shell
+$ nix-env -iA chain-utils-testnet -f https://github.com/crypto-com/chain-main/archive/f53418d87b8fede4fcee59078c996ee000eae4d2.tar.gz
+```
+
+After installation, you'll have these commands in your `PATH`:
+
+- `chain-maind`, the chain binary for testnet
+- `init-node`, the script to iniailize the chain data directory with state sync enabled
+- `print-systemd-config`, print a systemd config file to stdout
+
+## Initialization
+
+```shell
+$ export CHAINHOME=/path/to/data # optional, default to $HOME/.chain-maind
+$ MONIKER=testnode init-node
+```
+
+Change the value of `MONIKER` to your node's name.
+
+state sync is initialized and enabled by `init-node` automatically, if you don't want that, you can disable it by edit
+`$CHAINHOME/config/config.toml` manually, set `enable` field to `false` under section `[statesync]` like this:
+
+```
+[statesync]
+enable = false
+```
+
+## Run
+
+You can now run the chain node manually:
+
+```shell
+$ chain-maind start
+```
+
+Or setup systemd service on linux:
+
+```
+$ print-systemd-config | sudo tee /etc/systemd/system/chain-maind.service
+$ sudo systemctl start chain-maind
+```
+
+Please refer to [Croeseid Testnet](./croeseid-testnet.md#step-3-run-everything) for instructions to issue transactions
+and create validator.
+
+## Isolated installation
+
+One of the strength of nix package manager is you can isolated installation, it would be important when you want to manage both testnet and mainnet chains on single machine.
```

Comment:
```suggestion
One of the strengths of nix package manager is you can multiple isolated installations, it would be important when you want to manage both testnet and mainnet chains on a single machine.
```